release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -57,6 +57,18 @@ This release adds improvements related to the following components and concepts.
 
 [id="ocp-4-14-web-console"]
 === Web console
+
+[id="ocp-4-14-openshift-cli"]
+=== OpenShift CLI (oc)
+
+.oc-mirror supporting multi-arch OCI local images for catalogs
```

Comment:
I suggest reworking the title to start with something like "Support multi-arch OCI..."

Reply:
@tmalove How about "Support multi-arch OCI local images for catalogs with oc-mirror"?